### PR TITLE
fix carousel colors and height

### PR DESCRIPTION
**Pull request by SalischevAV**
fix carousel colors and height

### CODE REVIEW

**Review by letrylesha — APPROVED** [review]

File: packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcon.tsx [hunk 1]
```diff
@@ -38,6 +72,24 @@ function HouseIcon({house, setHoveredItem, hoveredItem}) {
...
+    if (!loading) {
+        if (!data.getPublicFlatsList.length) {
+            iconsColor = 'empty';
+        } else {
+            data.getPublicFlatsList.find((flat) => flat.status === 'RESERVED' || flat.status === 'FREE')
+                ? (iconsColor = 'free')
+                : (iconsColor = 'sold_out');
+        }
+    }
+
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcon.tsx:92]: simplify

File: packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcons.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import styled from 'styled-components';
 import Carousel from 'react-elastic-carousel';
 
 const HouseChooseContainer = styled.div`
+    height: 15vh;
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcons.tsx:8]: not sure you want 15vh.
Usually it is used as 100vh or in percents/pixels.

File: packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcon.tsx [hunk 3]
```diff
@@ -2,24 +2,58 @@ import ApartmentIcon from '@material-ui/icons/Apartment';
...
 `;
 
+function houseNameSplitter(houseName: string) {
+    const partArr = houseName.split(' ');
+    return (
+        <>
+            <div>
+                {partArr[0]} {partArr[1]}
+            </div>
+            <div>{partArr.splice(2, partArr.length - 2).join(' ')}</div>
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcon.tsx:52]: simplify

File: packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcon.tsx [hunk 4]
```diff
@@ -2,24 +2,58 @@ import ApartmentIcon from '@material-ui/icons/Apartment';
...
+        color: #808080;
+    }
+
+    &.free {
+        color: #008000;
+    }
+
+    &.sold_out {
+        color: #ff0047;
+    }
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcon.tsx:33]: Use BEM syntax for such cases. e.g `empty` status might be reserved by Mat UI

File: packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcon.tsx [hunk 5]
```diff
@@ -2,24 +2,58 @@ import ApartmentIcon from '@material-ui/icons/Apartment';
...
     flex-direction: column;
     align-items: center;
     padding: 12px;
     &:hover,
-    &.active {
+    &:hover *,
+    &.active,
+    &.active * {
         color: #e84f1d !important;
         cursor: pointer;
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/HouseIcon.tsx:18]: bad practice to use universal selectors

File: packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/ApartmentComplexLayout.tsx [hunk 6]
```diff
@@ -9,27 +9,25 @@ import {API_TOKEN} from '../../../core/constants';
...
-
-const LayoutContainer = styled.div`
-    height: 100vh;
 `;
 
 function attachSvg(container: string) {
     return SVG()
         .addTo(container)
-        .size('100%', '100%');
+        .size('100%', '80vh');
```
Comment (didevgen) [inline packages/raccoon-city-ui/src/components/Main/ApartmentComplexLayout/ApartmentComplexLayout.tsx:30]: don't use vh

**Review by didevgen — CHANGES_REQUESTED** [review]

**Pull request merged**